provision/aws/swarm/README.md

```diff
@@ -45,13 +45,14 @@ Separate 'master' and 'node' security groups control access between the nodes. T
 	- You can get the `ELBPublicDNSName` from cloudformation outputs as mentioned in above section. Outer proxy server acts as a bastion host.
 	- KeyName is the private key selected from the drop down list while creating the stack.
 - Clone ([this repository](https://github.com/Accenture/adop-docker-compose))
+- Run: cd adop-docker-compose
 - Run: export DOCKER\_HOST=tcp://\<SwarmMasterIP\>:2375
 	- You can get the `SwarmMasterIP` from cloudformation outputs as mentioned in above section.
-- Run: export TARGET\_HOST=\<ELBPublicDNSName\>
-	- You can get the `ELBPublicDNSName` from cloudformation outputs as mentioned in above section. 
+- Run: echo "export PROTO=https" > env.override.sh
```

Comment:
I'm thinking about the case, if i will add some env variables in env.override.sh in advance and after i will run ./adop compose init, this line will overwrite all my changes. Maybe it would better to use append operator ">>", what do you think?